twirl/src/main/scala/org/http4s/twirl/instances/package.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2014 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package twirl
+
+package object instances {
+  object all extends AllInstances
+  object twirl extends TwirlInstances
```

Comment:
Similar to the others, we could break these down into `Html`, `Xml`, etc.  Lots of boilerplate, but I suppose nice for people who like to keep their implicits as tight as possible.

Helps your commit stats, too. :laughing: